### PR TITLE
Update "Switching from TCP to HTTP/S" Doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
Add separate endpoint in synapse configuration.

### CODE REVIEW

File: en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md [hunk 1]
```diff
@@ -76,9 +89,9 @@ Send the following message via TCP to the TCP listener port.
     </soapenv:Body>
 </soapenv:Envelope>
 ``` 
-In Linux, we can save the above request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the above request in a <strong>request.xml</strong> file and use Netcat to send the TCP request. 
 ```
-netcat localhost 6060 < request.xml
+nc localhost 6060 < request.xml
```
Comment (chanikag) [inline en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md:94]: netcat and nc are same. why we need to change it?

File: en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md [hunk 2]
```diff
@@ -76,9 +89,9 @@ Send the following message via TCP to the TCP listener port.
     </soapenv:Body>
 </soapenv:Envelope>
 ``` 
-In Linux, we can save the above request in a <strong>request.xml</strong> file and use netcat to send the TCP request. 
+In Mac, we can save the above request in a <strong>request.xml</strong> file and use Netcat to send the TCP request.
```
Comment (chanikag) [inline en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md:92]: it is there in linux too. why we need to change it.

File: en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md (hunk 1 above)
Comment (chathuranga-jayanath-99) [reply en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md]: fixed it.

File: en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md (hunk 2 above)
Comment (chathuranga-jayanath-99) [reply en/docs/learn/examples/protocol-switching/switching-from-tcp-to-https.md]: fixed it.

**Comment by DinithiDiaz** [comment]
LGTM. @chanikag Can you please review this?

**Review by chanikag — APPROVED** [review]

**Pull request merged**